src/main/java/programminglife/model/exception/UnknownTypeException.java

```diff
@@ -0,0 +1,10 @@
+package programminglife.model.exception;
+
+/**
+ * Created by toinehartman on 04/05/2017.
+ */
+public class UnknownTypeException extends RuntimeException {
```

Comment:
So why would this be an unchecked exception instead of a checked exception?
I absolutely would not assume that the files don't have unknown symbols.